spec/c05recursion/generate/print_50_natural_nr_recursionSpec.js

```diff
@@ -0,0 +1,14 @@
+const printNaturalNumbers = require('../../../src/c05recursion/generate/print_50_natural_nr_recursion')
```

Comment:
print_50_natural_nr_recursion name is not correct it can print any number of numbers not only 50